src/main/java/com/minecolonies/coremod/network/messages/server/colony/building/TriggerSettingMessage.java

```diff
@@ -70,7 +70,7 @@ public void onExecute(
     {
         if (building.hasModule(SettingsModule.class))
         {
-            building.getFirstModuleOccurance(SettingsModule.class).ifPresent(m -> m.with(new SettingKey(this.value.getClass(), this.key), this.value));
+            building.getFirstModuleOccurance(SettingsModule.class).with(new SettingKey<>(this.value.getClass(), this.key), this.value);
         }
```

Comment:
Whereas this one used to be redundant and now isn't -- but it would be better to write it using `ifPresent` instead of `hasModule`.